webapp/routes_settings_radio.py

```
@@ -282,5 +313,55 @@ def api_radio_preset(preset_key: str) -> Any:
+
+            return jsonify({
+                "receiver_id": receiver_id,
+                "identifier": receiver.identifier,
+                "display_name": receiver.display_name,
+                "sample_rate": sample_rate,
+                "num_samples": num_samples,
+                "waveform": waveform_data,
+                "timestamp": time.time()
+            })
+
+        except Exception as exc:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)